packages/twenty-server/src/engine/core-modules/workspace/dtos/update-workspace-input.ts

```diff
@@ -12,135 +12,15 @@ import {
 } from 'class-validator';
 
 import { UUIDScalarType } from 'src/engine/api/graphql/workspace-schema-builder/graphql-types/scalars';
+import { RESERVED_SUBDOMAINS } from 'src/engine/core-modules/workspace/constants/reserved-subdomains.constant';
 
 @InputType()
 export class UpdateWorkspaceInput {
   @Field({ nullable: true })
   @IsString()
   @IsOptional()
   @Matches(/^(?!api-).*^[a-z0-9][a-z0-9-]{1,28}[a-z0-9]$/)
-  @IsNotIn([
```

Comment:
The automatedAI review raised a good point on casing, I would guess it affects updates too and there is an issue. Maybe we should get rid of Matches/IsNotIn and perform those checks in `updateWorkspace` method instead so that we can always force-convert to lowercase before?